ServerCore/Pages/Puzzles/Checklist.cshtml

```diff
@@ -61,7 +61,14 @@
                 <a asp-Page="./Edit" asp-route-puzzleid=@item.Puzzle.ID>@Html.DisplayFor(modelItem => item.Puzzle.Name)</a>
             </td>
             <td>
-                @Html.DisplayFor(modelItem => item.Puzzle.Group)
+                @if (item.Puzzle.Group != null && item.Puzzle.Group.StartsWith("Html.Raw(") && item.Puzzle.Group.EndsWith(")"))
```

Comment:
I hear you, and that's certainly a workable proposal, but we already have the Html.Raw convention elsewhere (in Responses, so authors can put links in response text where useful) and I'd rather not have two competing raw HTML conventions unless we really have to. [Long term, I'd like to pitch the raw HTML convention entirely and just move to markdown, but I'm nervous about picking the right library for it.]

Before making the change, I thought about the sorting issue you mention, but I eventually assumed that any event that made use of this convention would just use Html.Raw on all groups and then it would sort well enough.

So: If the code as written is sufficient for PH22, I propose to leave it as is and eventually replace all raw HTML support with markdown. But if it's insufficient for PH22, I can/will change it.

Reply:
This would require every contributor to use Html.Raw if just one wants to use it and all use a common structure that enables sorting, which would be even harder with more independent constructing teams (we just have three in PH22). How about this variation instead:
```
@if (item.Group != null && item.Puzzle.Group.EndsWith(")")
    && (item.Group.StartsWith("Html.Raw(") || item.Group.Contains("::Html.Raw(")))
{
    int start = item.Puzzle.group.Index("Html.Raw(") + 9;
    @Html.Raw(item.Puzzle.Group.Substring(start, item.Puzzle.Group.Length - start - 1));
}
else
{
    @Html.DisplayFor(modelItem => item.Group)
}
```

Reply:
Makes sense - but I'll just trim to this:

```
@if (item.Group != null && item.Group.Contains("Html.Raw(")) && item.Puzzle.Group.EndsWith(")"))
{
    int start = item.Puzzle.group.Index("Html.Raw(") + 9;
    @Html.Raw(item.Puzzle.Group.Substring(start, item.Puzzle.Group.Length - start - 1));
}
else
{
    @Html.DisplayFor(modelItem => item.Group)
}
```

I'll put this in a new PR since this one is already merged. Thanks for the idea!